.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test Python ${{ matrix.python-version }}
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.6'
+          - '3.7'
+          - '3.8'
+          - '3.9'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-python-${{ matrix.python-version }}-${{ hashFiles('**/setup.*', '**/tox.ini', '**/requirements*.txt') }}
+          restore-keys: |
+            pip-python-${{ matrix.python-version }}-
+            pip-python-
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade tox tox-gh-actions
+
+      - name: Tox tests
+        run: |
+          tox -- --cov=pyuwsgimemhog --cov-report xml
+
+      - name: Upload coverage
```

Comment:
please drop codecov upload and just print the textual representation on the console output